app/views/providers/means_reports/_with_cfe_result_details.html.erb

```diff
@@ -90,13 +86,11 @@
 <% end %>
 
 <% if @legal_aid_application.uploading_bank_statements? %>
-<div class="govuk-!-padding-bottom-8"></div>
+  <section class="print-no-break">
+    <h3 class="govuk-heading-m"><%= t(".bank_statement_heading") %></h3>
 
-<section class="print-no-break">
-  <h3 class="govuk-heading-m"><%= t(".bank_statement_heading") %></h3>
-  <% if @legal_aid_application.uploading_bank_statements? %>
     <% has_partner = @legal_aid_application.applicant.has_partner_with_no_contrary_interest? %>
-    <%= govuk_summary_list(actions: false, classes: "govuk-!-margin-bottom-9") do |summary_list| %>
+    <%= govuk_summary_list(card: { title: t(".bank_statement_heading") }, actions: false) do |summary_list| %>
```

Comment:
Non blocking question... does adding the title value change the bottom margin or do the new designs call for sections to be closer together?

Reply:
Hmm, just seen the next page still has the ```classes: "govuk-!-margin-bottom-9"``` so perhaps this was an oversight?